### PR TITLE
Fix edge_count.yml benchmark name.

### PR DESCRIPTION
**Pull request by filipecosta90**
The space in the test name is making the grafana redis datasource not behaving as expected ( making us unable to chart this use-case ). The solution is simple - rename the test so that it does not contain spaces.

### CODE REVIEW

File: tests/benchmarks/entity_count.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
-name: "ENTITY COUNT"
+name: "ENTITY_COUNT"
 remote:
   - setup: redisgraph-r5
   - type: oss-standalone
 dbconfig:
   - init_commands:
-    - "GRAPH.QUERY g UNWIND range(0, 5000000) AS x CREATE (:N)-[:R]->(:N)"
+    - '"GRAPH.QUERY" "g" "UNWIND range(0, 5000000) AS x CREATE (:N)-[:R]->(:N)"'
```
Comment (swilly22) [inline tests/benchmarks/entity_count.yml:7]: was this change necessary?

Comment (filipecosta90) [reply tests/benchmarks/entity_count.yml:7]: Yes, given we have spaces in the same argument ( and to make the tool agnostic ) we need the `"`

**Review by swilly22 — APPROVED** [review]
